oh_staff_ui/management/commands/create_mods_records.py

```diff
@@ -7,29 +7,63 @@
 logger = logging.getLogger(__name__)
 
 
-def create_mods_records(item_id: int) -> None:
-    """Given an item_id write the related MODS record to public location"""
-    try:
-        pi = ProjectItem.objects.get(id=item_id)
-        OralHistoryMods(pi).write_mods_record()
-    except (ProjectItem.DoesNotExist) as e:
-        logger.error(e)
-        raise CommandError(f"ProjectItem with id {item_id} does not exist")
-
-
 class Command(BaseCommand):
     help = "Django management command to generate Oral History MODS records"
 
     def add_arguments(self, parser):
         parser.add_argument(
             "-i",
             "--item_id",
+            nargs="+",
             type=int,
-            required=True,
-            help="The id of the item to export mods record of",
+            required=False,
+            help="The id(s) of the item to export mods record of",
+        )
+        parser.add_argument(
+            "-b",
+            "--bulk",
+            type=str,
+            required=False,
+            choices=["series", "interview", "audio", "all"],
+            help="The type of items to bulk export, choices are: series, interview, assets and all",
         )
 
     def handle(self, *args, **options):
 
-        item_id = options["item_id"]
-        create_mods_records(item_id)
+        if options["item_id"]:
+            for item_id in options["item_id"]:
```

Comment:
See previous comment about supporting multiple single item exports.  If this is desired, please refactor so the driver for this can reuse the `try/except` and the `count` output message currently in the bulk export.